src/DssSpell.t.sol

```diff
@@ -904,63 +908,368 @@ contract DssSpellTest is DssSpellTestBase {
+            addr.addr("RWA014_A_OUTPUT_CONDUIT"),
+            addr.addr("RWA007_A_JAR_INPUT_CONDUIT"),
+            addr.addr("RWA007_A_INPUT_CONDUIT"),
+            addr.addr("RWA007_A_OUTPUT_CONDUIT"),
+            addr.addr("RWA015_A_INPUT_CONDUIT_JAR_USDC"),
+            addr.addr("RWA015_A_INPUT_CONDUIT_URN_USDC"),
+            addr.addr("RWA009_A_INPUT_CONDUIT_URN_USDC")
+        ];
 
-        uint256 numJobs = DssCronSequencerLike(CRON_SEQUENCER).numJobs();
+        // ----- Pre-spell sanity checks -----
+        // single swap conduits
+        _checkSingleSwapRwaConduits(singleSwapConduits, MCD_PSM_USDC_A);
 
-        assertEq(DssCronSequencerLike(CRON_SEQUENCER).hasJob(CRON_CLIPPER_MOM_JOB) , true, "TestError/old-job-not-present-in-sequencer");
-        assertEq(DssCronSequencerLike(CRON_SEQUENCER).hasJob(CRON_CLIPPER_MOM_JOB_NEW), false, "TestError/new-job-already-present-in-sequencer");
+        // multi swap conduit
+        assertEq (RwaConduitLike(RWA015_A_OUTPUT_CONDUIT).pal(MCD_PSM_USDC_A), 1);
+        assertEq (RwaConduitLike(RWA015_A_OUTPUT_CONDUIT).pal(MCD_LITE_PSM_USDC_A), 0);
 
+        // ----- Execute spell -----
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done(), "TestError/spell-not-done");
 
-        assertEq(DssCronSequencerLike(CRON_SEQUENCER).hasJob(CRON_CLIPPER_MOM_JOB), false, "TestError/old-job-not-removed-from-sequencer");
-        assertEq(DssCronSequencerLike(CRON_SEQUENCER).hasJob(CRON_CLIPPER_MOM_JOB_NEW), true, "TestError/new-job-not-added-to-sequencer");
-        assertEq(DssCronSequencerLike(CRON_SEQUENCER).numJobs(), numJobs, "TestError/job-amount-changed");
+        // ----- Post-spell state checks -----
+        // single swap conduits
+        _checkSingleSwapRwaConduits(singleSwapConduits, MCD_LITE_PSM_USDC_A);
+
+        // multi swap conduit
+        assertEq (RwaConduitLike(RWA015_A_OUTPUT_CONDUIT).pal(MCD_PSM_USDC_A), 0);
+        assertEq (RwaConduitLike(RWA015_A_OUTPUT_CONDUIT).pal(MCD_LITE_PSM_USDC_A), 1);
+    }
+
+    function _checkSingleSwapRwaConduits(address[9] memory conduits, address psm) internal {
+        for (uint256 i; i < conduits.length - 1; i++) {
+            assertEq(
+                RwaConduitLike(conduits[i]).psm(),
+                psm
+            );
+        }
+    }
+
+    function test_LITE_PSM_USDC_A_MigrationPhase2() public {
+        (uint256 psrcInk, uint256 psrcArt) = vat.urns(SRC_ILK, address(srcPsm));
+        uint256 psrcVatGem = vat.gem(SRC_ILK, address(srcPsm));
+        uint256 psrcGemBalance = gem.balanceOf(address(srcPsm.gemJoin()));
+        (uint256 pdstInk, uint256 pdstArt) = vat.urns(DST_ILK, address(dstPsm));
+        uint256 pdstVatGem = vat.gem(DST_ILK, address(dstPsm));
+        uint256 pdstGemBalance = gem.balanceOf(address(pocket));
+
+        uint256 expectedMoveWad = _min(psrcInk, _subcap(psrcInk, srcKeep));
+
+        // ----- Pre-spell sanity checks -----
```

Comment:
Nit: ideally, you would also check `srcPsm.tin()`, `srcPsm.tout()` and `dstPsm.buf()` before the spell (to ensure correctness of the instructions such as "PSM-USDC-A tin: Increase by 0.01 percentage points, __from 0%__ to 0.01%")

Reply:
Done in [65f90b2](https://github.com/makerdao/spells-mainnet/pull/422/commits/65f90b2d5e9397b42f8014ad78f58006e1e4535d)